cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/ApplyController.java

```diff
@@ -81,22 +82,29 @@ public Integer call() throws Exception {
         @Mixin
         private YamlCommandOptions yamlCommandOptions;
 
+
+        @Option(names = { "-ns", "--no-auto-start" }, required = false,
+                description = "Deployed apps won't get started automatically.")
+        private boolean noAutoStart;
+
         @Override
         public Integer call() throws Exception {
-            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
-
-            ApplyLogic applyLogic = new ApplyLogic(cfOperations);
 
             log.info("Interpreting YAML file...");
             ConfigBean desiredConfigBean = YamlMapper.loadBeanFromFile(yamlCommandOptions.getYamlFilePath(),
                     ConfigBean.class);
             log.verbose("YAML file interpreted.");
 
-            if(desiredConfigBean.getSpec() == null || desiredConfigBean.getSpec().getApps() == null) {
+            if (desiredConfigBean.getSpec() == null || desiredConfigBean.getSpec().getApps() == null) {
                 log.info("No apps node specified in the yaml file. Nothing to apply...");
                 return 0;
             }
 
+            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+
+            log.info("Auto starting apps:", !noAutoStart);
```

Comment:
I would recommend to use `verbose` logging here, because normally the user should be aware of that fact

Reply:
Agree